R/texreg.R

```diff
@@ -113,12 +113,28 @@ matrixreg <- function(l,
   }
 
   # grouping
-  output.matrix <- grouping(output.matrix, groups, indentation = "    ", 
-      single.row = single.row, prefix = "", suffix = "")
+  if (dots$output.type == 'ascii') {
+      output.matrix <- grouping(output.matrix, groups, indentation = "    ", 
+                                single.row = single.row, prefix = "", suffix = "", rowLabelType = 'text')
+  } else if (dots$output.type == 'latex') {
+      output.matrix <- grouping(output.matrix, groups, indentation = "    ", 
+                                single.row = single.row, prefix = "", suffix = "", rowLabelType = 'latex')
+  }    else if (dots$output.type == 'html') {
+      output.matrix <- grouping(output.matrix, groups, indentation = "    ", 
+                                single.row = single.row, prefix = "", suffix = "", rowLabelType = 'text')
+  }
```

Comment:
I think these conditions are overly complicated. Do this as the initial `if` condition: `if (dots$output.type == 'latex') {`, and then the text case can just be done using `else` instead of `else if`. That leaves us with only two instead of three statements. OK?